views/_includes/input-dates.pug

```diff
@@ -0,0 +1,28 @@
+mixin dateInputs(idPrefix, legendText, isoHintText, values)
+  div.date-container(class={
+    ['has-error--container']: errors,
+    ['has-error']: errors && errors[`${idPrefix}Day`] && errors[`${idPrefix}Day`].msg === 'errors.login.dateOfBirth.match'
+    })
+    fieldset.fieldset(role="group", aria-describedby=`${idPrefix}Hint`,)
+      legend
+        h2 #{__(legendText)}
+
+      span#dobHint.cra-form-message #{__('For Example:')} #{isoDateHintText(isoHintText)}
+
+      if errors && errors[`${idPrefix}Day`] || errors && errors[`${idPrefix}Month`] || errors && errors[`${idPrefix}Day`]
+        ul
+          each err in errors
+            if err.param.includes(idPrefix)
+              li
+                +validationMessage(err.msg, err.param)
+      div.w-1-4.d--ib(class={['has-error--inline']: errors && errors[`${idPrefix}Day`]})
+        label(for=`${idPrefix}Day`) #{__('Day')}
+        input#dobDay(name=`${idPrefix}Day`, autofocus, type='text', value=values[0], autocomplete=`${idPrefix}-day`, aria-describedby=(errors && errors[`${idPrefix}Day`] ? `${idPrefix}Day-error` : false))
+
+      div.w-1-4.d--ib(class={['has-error--inline']: errors && errors[`${idPrefix}Month`]})
+        label(for=`${idPrefix}Month`) #{__('Month')}
+        input#dobMonth(name=`${idPrefix}Month`, type='text', value=values[1], autocomplete=`${idPrefix}-month`, aria-describedby=(errors && errors[`${idPrefix}Month`] ? `${idPrefix}Month-error` : false))
```

Comment:
This autocomplete one is a bit weird: 
https://design-system.service.gov.uk/components/date-input/#use-the-autocomplete-attribute-for-a-date-of-birth

Basically, if we're asking for a date of birth for the current user, we should include this, and then hardcode the values (as `bday-day`, `bday-month`, `bday-year`). Otherwise, we shouldn't show the autcomplete at all. We could use the `dob` idPrefix to decide, or another var (defaults to false).

We could also drop support for this and circle back later.